[2025-03-17T22:30:26.385+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T17:51:08.903472+00:00 [queued]>
[2025-03-17T22:30:26.577+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T17:51:08.903472+00:00 [queued]>
[2025-03-17T22:30:26.578+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-17T22:30:27.837+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): transform_data_phase_1_task> on 2025-03-16 17:51:08.903472+00:00
[2025-03-17T22:30:27.889+0000] {standard_task_runner.py:60} INFO - Started process 15986 to run task
[2025-03-17T22:30:28.003+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Google_map', 'transform_data_phase_1_task', 'manual__2025-03-16T17:51:08.903472+00:00', '--job-id', '195', '--raw', '--subdir', 'DAGS_FOLDER/google_map_dag_etl.py', '--cfg-path', '/tmp/tmpagce9ozn']
[2025-03-17T22:30:28.066+0000] {standard_task_runner.py:88} INFO - Job 195: Subtask transform_data_phase_1_task
[2025-03-17T22:30:31.508+0000] {task_command.py:423} INFO - Running <TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T17:51:08.903472+00:00 [running]> on host DESKTOP-AVAHALM.
[2025-03-17T22:30:35.849+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='master_m2si' AIRFLOW_CTX_DAG_ID='Google_map' AIRFLOW_CTX_TASK_ID='transform_data_phase_1_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T17:51:08.903472+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T17:51:08.903472+00:00'
[2025-03-17T22:30:35.855+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-17T22:30:35.856+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/data_warehouse_project && dbt run --profiles-dir ~/.dbt --models cleaned_reviews']
[2025-03-17T22:30:35.948+0000] {subprocess.py:86} INFO - Output:
[2025-03-17T22:31:00.333+0000] {subprocess.py:93} INFO - [0m22:31:00  Running with dbt=1.9.0-b2
[2025-03-17T22:31:06.704+0000] {subprocess.py:93} INFO - [0m22:31:06  Registered adapter: postgres=1.8.2
[2025-03-17T22:31:10.599+0000] {subprocess.py:93} INFO - [0m22:31:10  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-17T22:31:10.600+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-17T22:31:10.645+0000] {subprocess.py:93} INFO - - models.data_warehouse_project.example
[2025-03-17T22:31:13.382+0000] {subprocess.py:93} INFO - [0m22:31:13  Found 6 models, 1 source, 424 macros
[2025-03-17T22:31:13.393+0000] {subprocess.py:93} INFO - [0m22:31:13
[2025-03-17T22:31:13.399+0000] {subprocess.py:93} INFO - [0m22:31:13  Concurrency: 4 threads (target='dev')
[2025-03-17T22:31:13.401+0000] {subprocess.py:93} INFO - [0m22:31:13
[2025-03-17T22:31:15.734+0000] {subprocess.py:93} INFO - [0m22:31:15  1 of 1 START sql table model public.cleaned_reviews ............................ [RUN]
[2025-03-17T22:31:18.867+0000] {subprocess.py:93} INFO - [0m22:31:18  1 of 1 OK created sql table model public.cleaned_reviews ....................... [[32mSELECT 7[0m in 3.12s]
[2025-03-17T22:31:19.254+0000] {subprocess.py:93} INFO - [0m22:31:19
[2025-03-17T22:31:19.268+0000] {subprocess.py:93} INFO - [0m22:31:19  Finished running 1 table model in 0 hours 0 minutes and 5.85 seconds (5.85s).
[2025-03-17T22:31:19.955+0000] {subprocess.py:93} INFO - [0m22:31:19
[2025-03-17T22:31:19.970+0000] {subprocess.py:93} INFO - [0m22:31:19  [32mCompleted successfully[0m
[2025-03-17T22:31:20.021+0000] {subprocess.py:93} INFO - [0m22:31:20
[2025-03-17T22:31:20.023+0000] {subprocess.py:93} INFO - [0m22:31:20  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-17T22:31:23.894+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-17T22:31:24.783+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Google_map, task_id=transform_data_phase_1_task, execution_date=20250316T175108, start_date=20250317T223026, end_date=20250317T223124
[2025-03-17T22:31:25.231+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T22:31:25.741+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
